### DISCUSSION TITLE
STA lib info marshalling/unmarshalling

### DISCUSSION THREAD

**Discussion by donn** [The-OpenROAD-Project/OpenROAD#5677] [Ideas]
Currently, data has to be read from `.lib` files every time. This has been fine for open-source SCLs, but some proprietary SCLs take 5-10 seconds to load.

Proprietary tools solve this by compiling/marshalling the library info for faster loading later. Has doing this in OpenROAD/OpenSTA been discussed?

**Comment by maliberty** [comment]
No but CCS might push us in that direction.  Is that what you are using?
